### ISSUE TITLE
Having problems installing Second Wind

### ISSUE THREAD

**Issue by ferrazpedro** [NiceneNerd/ukmm#134]
Hey guys! I'm having some problems installing the Second Wind mod. Every time I try to import it into UKMM it gives me this error:

`Failed to convert BNP to UKMM mod

Caused by:
    0: Failed to package mod root at C:\Users\pedro\AppData\Local\Temp\.tmpQb3oUu for mod Second Wind v1.9.13 at `crates\uk-mod\src\pack.rs@605:42`
    1: Failed to process SARC file Actor/Pack/ElementalRodAttack.bactorpack at `crates\uk-mod\src\pack.rs@378:22`
    2: Failed to parse resource Actor/AIProgram/ElementalRodAttack.baiprog in SARC content/Actor/Pack/ElementalRodAttack.sbactorpack at `crates\uk-mod\src\pack.rs@502:73`
    3: Failed to parse resource Actor/AIProgram/ElementalRodAttack.baiprog
    4: Failed to collect AI program tree roots
    5: Parameter file missing key: AI program missing Query_13 as child of Root`

**Comment by NiceneNerd** [comment]
Duplicate of several other issues. We're waiting on a fix from Second Wind.
